src/Microsoft.DocAsCode.Build.Engine/PostProcessors/HtmlPostProcessor.cs

```diff
@@ -32,7 +32,21 @@ public string GetIncrementalContextHash()
             if (!_handlerInitialized)
             {
                 Handlers.Add(new ValidateBookmark());
-                if (!metadata.ContainsKey("_keepDebugInfo") || ((bool)metadata["_keepDebugInfo"]) == false)
+
+                bool keepDebugInfo = false;
+                // get keepDebugInfo from environment
+                var docfxKeepDebugInfo = Environment.GetEnvironmentVariable("DOCFX_KEEP_DEBUG_INFO");
+                if (!string.IsNullOrEmpty(docfxKeepDebugInfo) && bool.TryParse(docfxKeepDebugInfo, out keepDebugInfo))
+                {
+                    Logger.LogInfo($"DOCFX_KEEP_DEBUG_INFO is set to {keepDebugInfo}");
+                }
+                // get keepDebugInfo from docfx config globalMetadata
+                if (metadata.ContainsKey("_keepDebugInfo"))
```

Comment:
After second thoughts, we can skip metadata check here. This logic is used in another feature "inline editing", but this feature is no longer needed.